server/src/test/java/org/elasticsearch/action/support/AutoCreateIndexTests.java

```diff
@@ -201,8 +210,9 @@ private static ClusterState buildClusterState(String... indices) {
     }
 
     private AutoCreateIndex newAutoCreateIndex(Settings settings) {
+        SystemIndices systemIndices = new SystemIndices(Map.of("plugin", List.of(new SystemIndexDescriptor(".foo", ""))));
```

Comment:
Nitpick: I'd recommend changing the name some something like `.test-system-index`, just to make it a bit more immediately obvious what's going on rather than using a generic `.foo` here because this is shared across tests.

Reply:
Good call. I'll make it a constant too so that it's easier to find usages.